link-grammar/prepare/build-disjuncts.c

```diff
@@ -13,22 +13,19 @@
 
 /* stuff for transforming a dictionary entry into a disjunct list */
 
-#include <math.h>
 #include "build-disjuncts.h"
 #include "connectors.h"
-//#include "dict-common/dict-api.h"        // for print_expression
-#include "dict-common/dict-structures.h"   // for Exp_struct
+//#include "dict-common/print-dict.h"      // print_expression()
+#include "dict-common/dict-structures.h"   // Exp_struct()
 #include "disjunct-utils.h"
 #include "utilities.h"
 
 /* Temporary connectors used while converting expressions into disjunct lists */
 typedef struct Tconnector_struct Tconnector;
 struct Tconnector_struct
 {
-	char multi;   /* TRUE if this is a multi-connector */
-	char dir;     /* '-' for left and '+' for right */
 	Tconnector * next;
-	const condesc_t * condesc;
+	const Exp *e;
```

Comment:
Hmm. This is now a strange and confusing name. The old `Tconnectors` were obviously connectors: a multi, a dir, a connector-struct. The new one is .. a linked-list of expressions.

I guess that each expression in the list describes exactly one connector, right? There should be a note that explains this.  A name-change would be good, also: rename `Tconnector` to `SingleConnectorExp`  or something like that.

Reply:
Or 'ConnectorPerExp` or `ConnectorInExp` or ... or maybe we can leave it as Tconnector, as long as there's an explanation of the expected contents.

Reply:
> I guess that each expression in the list describes exactly one connector, right? There should be a note that explains this.

Yes.
> .. or maybe we can leave it as Tconnector, as long as there's an explanation of the expected contents.

I will add an explanation.